sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerProviderBuilder.java

```diff
@@ -5,13 +5,14 @@
 
 package io.opentelemetry.sdk.trace;
 
+import static java.util.Objects.requireNonNull;
```

Comment:
Intentional change?

Reply:
I needed to add `"traceConfig"` to the requireNonNull check for it and used my normal static import style. If it was 100 lines of drift I'd probably revert it but hoping just three is ok :)